docs/source/yelpsoa_configs.rst

```diff
@@ -60,22 +60,25 @@ specify the following options:
   * ``env``: A dictionary of environment variables that will be made available
     to the container. PaaSTA additionally will inject the following variables automatically (keep in mind all environment variables are strings in a shell):
 
-    * ``PAASTA_SERVICE``: The service name
-    * ``PAASTA_INSTANCE``: The instance name
+    * ``MESH_REGISTRATIONS`` (optional): space-separated list of registrations for a given Pod if the PaaSTA instance is mesh-routable - absent otherwise.
     * ``PAASTA_CLUSTER``: The cluster name
-    * ``PAASTA_HOST``: The hostname of the actual server the container is runnig on
-    * ``PAASTA_PORT``: The configured port the service should listen on
+    * ``PAASTA_DEPLOY_GROUP``: The `deploy group <deploy_group.html>`_ specified
     * ``PAASTA_DOCKER_IMAGE``: The docker image name
     * ``PAASTA_GIT_SHA``: The short git sha of the code the container has
-    * ``PAASTA_DEPLOY_GROUP``: The `deploy group <deploy_group.html>`_ specified
+    * ``PAASTA_HOST``: The hostname of the actual server the container is runnig on
+    * ``PAASTA_IMAGE_VERSION``: The version of the docker image
```

Comment:
I think this too is not always included, only for those using `--image-version` w/ mark-for-deployment (likely only those using `enable_automated_redeploys` in yelpsoa w/ our standard pipelines)

Reply:
updated!